quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/dashboard/DashboardMapper.java

```diff
@@ -6,10 +6,11 @@
 import com.github.reflectoring.haljson.HalJsonResource;
 import com.github.reflectoring.haljson.Link;
 import com.github.reflectoring.haljson.ResourceMapper;
+import com.github.reflectoring.infiniboard.packrat.dashboard.DashboardConfig;
 import com.github.reflectoring.infiniboard.quartermaster.widget.WidgetConfigMapper;
 
 @Component
-public class DashboardMapper implements ResourceMapper<Dashboard> {
+public class DashboardMapper implements ResourceMapper<DashboardConfig> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)